marc.py

```diff
@@ -618,9 +610,10 @@ def records(self, lane, annotator, start_time=None, force_refresh=False,
         # and Representation, but don't actually mirror it.
         if not mirror:
             storage_protocol = self.integration.setting(self.STORAGE_PROTOCOL).value
-            mirror = MirrorUploader.sitewide(self._db)
+            # todo mirror
+            mirror = MirrorUploader.mirror(self._db, storage_name)
```

Comment:
The only place this function `records` is called is in scripts.py `CacheMARCFiles` but I'm not sure how to get the name of the collection or storage name to pass it to `.records()`.

Reply:
If you have a configuration setting called STORAGE_INTEGRATION_ID (or whatever) you can load the ExternalIntegration by its database ID and then pass the ExternalIntegration into MirrorUplaoder.implementation().